components/event-sources/upgrade-job/process/add_steps.go

```diff
@@ -0,0 +1,16 @@
+package process
+
+func (p *Process) AddSteps() {
+	p.Steps = []Step{
+		NewSaveCurrentState(p),
+		NewDeleteEventServiceDeployments(p),
+		NewDeleteEventSources(p),
+		NewUnLabelNamespace(p),
+		NewDeleteBrokers(p),
+		NewConvertTriggersToKymaSubscriptions(p),
+		NewDeleteTriggers(p),
+		NewDeleteAppSubscriptions(p),
+		NewPatchConnectivityValidators(p),
+		NewCleanUp(p),
```

Comment:
Should `channels` be deleted as well?

Reply:
Channels are deleted through 2 actions(which I already took care):
- deletion of HTTPSources
- deletion of Brokers
Are there any specific channels that will be left over?